macstadium-shared-1/Makefile

```diff
@@ -4,27 +4,29 @@ include $(shell git rev-parse --show-toplevel)/terraform.mk
 default: hello
 
 CONFIG_FILES := \
-		config/jupiter-brain-prod-org-env \
+		config/jupiter-brain-production-org-env \
 		config/jupiter-brain-staging-org-env \
 		config/jupiter-brain-custom-1-env \
 		config/jupiter-brain-custom-2-env \
 		config/jupiter-brain-custom-3-env \
 		config/travis-worker-org-staging-1 \
 		config/travis-worker-org-staging-2 \
-		config/travis-worker-org-prod-1 \
-		config/travis-worker-org-prod-2 \
+		config/travis-worker-org-production-1 \
+		config/travis-worker-org-production-2 \
 		config/travis-worker-custom-1 \
 		config/travis-worker-custom-2 \
 		config/travis-worker-custom-3 \
-		config/vsphere-janitor-prod-com \
+		config/vsphere-janitor-production-com \
 		config/vsphere-janitor-staging-com \
 		config/vsphere-janitor-custom-1 \
 		config/vsphere-janitor-custom-2 \
 		config/vsphere-janitor-custom-3 \
-		config/jupiter-brain-prod-com-env \
+		config/jupiter-brain-production-com-env \
 		config/jupiter-brain-staging-com-env \
+		config/travis-worker-com-production-1 \
+		config/travis-worker-com-production-2 \
 		config/travis-worker-com-staging-1 \
-		config/travis-worker-com-prod-1 \
+		config/travis-worker-com-staging-2 \
```

Comment:
fyi, staging-2 was missing here, and prod-1 was renamed to production and moved up